_examples/document/doc-custom-properties/main.go

```diff
@@ -31,4 +32,15 @@ func main() {
 	cp.SetPropertyAsDate("Another date property", time.Now()) // date
 
 	doc.SaveToFile("document.docx")
```

Comment:
change this output to document_customized.docx (avoid overwriting original  document.docx).